src/components/FilterBar.jsx

```diff
@@ -1,112 +1,72 @@
   const [anchorEl, setAnchorEl] = useState(null);
 
   return (
-    <>
-      <Button
-        variant="outlined"
-        onClick={(e) => setAnchorEl(e.currentTarget)}
-        endIcon={<ArrowDropDownIcon color="primary" />}
-        sx={{ m: 1, height: (theme) => theme.spacing(4) }}>
-        {title}
-      </Button>
-      <Popover
-        open={Boolean(anchorEl)}
-        anchorEl={anchorEl}
-        onClose={() => setAnchorEl(null)}
-        anchorOrigin={{ vertical: 'bottom', horizontal: 'left' }}
-        transformOrigin={{ vertical: 'top', horizontal: 'left' }}
-        PaperProps={{ style: popoverWidth }}>
-        {children}
-      </Popover>
-    </>
-  );
-};
-
-FilterButton.propTypes = {
-  title: PropTypes.string.isRequired,
-  children: PropTypes.node.isRequired,
-  popoverWidth: PropTypes.object,
-};
-
-FilterButton.defaultProps = {
-  popoverWidth: undefined,
-};
-
-export default function FilterBar() {
-  const [{ minAmount, maxAmount, grades, majors, sortBy }, setQueryParam] =
-    useQueryParams();
-
-  return (
-    <Grid
-      container
-      spacing={2}
-      justifyContent="space-between"
-      sx={{ flexGrow: 1 }}>
-      <Grid item>
-        <FilterButton title="Majors" popoverWidth={{ minWidth: 280 }}>
-          <MajorFilter
-            majors={majors}
-            onSelect={(m) => setQueryParam('majors', m)}
-            onDelete={(m) =>
-              setQueryParam(
-                'majors',
-                majors.filter((major) => major !== m)
-              )
-            }
-          />
-        </FilterButton>
+    <Box sx={{ pb: 3 }}>
+      <Grid
+        container
+        justifyContent="space-between"
+        alignItems="center"
+        direction="row-reverse"
+        sx={{ flexGrow: 1, bgcolor: 'primary.main', p: 2 }}>
+        <Grid item sx={{ display: 'flex', alignItems: 'center' }}>
+          <IconButton
+            onClick={(e) => setAnchorEl(e.currentTarget)}
+            color="secondary">
+            <ImportExportIcon />
+          </IconButton>
+          <Typography color="secondary">Sort</Typography>
+        </Grid>
 
-        <FilterButton title="Grades">
-          <GradeLevelFilter
-            grades={new Set(grades)}
-            changeFn={(e) => setQueryParam('grades', e)}
-          />
-        </FilterButton>
+        <Grid
+          item
+          sx={{ display: { xs: 'flex', md: 'none' }, alignItems: 'center' }}>
+          <IconButton onClick={openFilter} color="secondary">
+            <TuneIcon />
+          </IconButton>
 
-        <FilterButton title="Amount">
-          <AmountFilter
-            min={minAmount ?? 0}
-            max={maxAmount ?? 0}
-            onMinChange={(e) => setQueryParam('minAmount', e.target.value)}
-            onMaxChange={(e) => setQueryParam('maxAmount', e.target.value)}
-          />
-        </FilterButton>
+          <Typography color="secondary">Filter</Typography>
+        </Grid>
```

Comment:
Again I think a [button with an icon](https://mui.com/components/buttons/#buttons-with-icons-and-label) keeps this simple.